scripts/publish-function.sh

```diff
@@ -38,7 +38,7 @@ queueName=${botName//_/-}
 echo "About to publish function ${functionName}"
 gcloud functions deploy "${functionName}" \
   --trigger-http \
-  --runtime nodejs10 \
+  --runtime nodejs12 \
```

Comment:
How about to parameterize this like 

```
--runtime "${functionRuntime}" \
```

Then in the bot directory:

```
if [[ -f .env_deployment ]]; then
    source .env_deployment
fi

if [[ -z "${functionRuntime:-}" ]]; then
    # defaults to nodejs10
    functionRuntime=nodejs10
fi 
```
or something?

Reply:
Or maybe we can determine the nodejs runtime by grep against package.json .